### ISSUE TITLE
click-casting breaks incoming heals?

### ISSUE THREAD

**Issue by mihneaadr** [Aviana/LunaUnitFrames#146]
I've isolated the problem over the course of 5 hours of troubleshooting. For some reason, certain bindings break incoming heals in a very random, yet oddly specific (as in limited) way. As far as I can see, binding healing touch on my Left click bugs it AND ONLY it, i.e. casting regrowth with shift-left-click still shows inc heals, and casting from the bar ALWAYS works properly; binding something to MB4 breaks it, and other such oddly limited things. This occurs during the actual binding, in the menu, and is sometimes randomly fixed when alt-tabbing or casting a certain random sequence of spells (or, at least, one which I haven't been able to figure out the rules for). To be completely honest I don't even know how to describe it properly anymore than "After I set up my click-casting shortcuts, the incoming heals feature sometimes works, sometimes doesn't."...

**Comment by Aviana** [comment]
Just tested on a lvl 4 priest. Bound all mouse buttons to lesser heal. All incoming heals show up. Try disabling all other addons and see if the problem persists.

**Comment by mihneaadr** [comment]
All addons except Luna have been disabled. Still the same bugs.

**Comment by mihneaadr** [comment]
So I've kept testing this issue on 3 different machines: my old, crappy desktop; my SO's laptop and my own new laptop (all of these tests were done using the same mouse). Funnily enough, the desktop is the only one which has no problems running the add-on properly, as intended. Meanwhile on the 2 RoG laptops that I've tested this on (one running windows 7, one windows 10) it rarely seemed to work properly. I'm thinking it might have something to do with either the fact that laptops basically have 2 sets of mouse input (the pad, and the actual mouse I plugged in), or with the Asus smart gesture drivers. Starting the game with the external mouse plugged out seems to resolve the problems for a while, with the click-casting part working as intended (although eventually it reverts to the behavior I previously described, and I cannot, for the life of me, tell you what triggers it), however disconnecting and reconnecting the mouse does nothing if click-casting is already broken. Thought these details might help.
